server/lib/template.js

```diff
@@ -130,4 +133,21 @@ function lowercase(value) {
   return String(value).toLowerCase();
 }
 
+/**
+ * @function sum
+ * @desc Summation by column
+ * @param {array} array The array of element
+ * @param {string} column The column of summation
+ * @return {number} the summation
+ */
+function sum(array, column, ponderation) {
+  ponderation = typeof(ponderation) === 'string' ? ponderation : null;
+
+  if (!array || (array && !array.length)) { return; }
+
+  return array.reduce(function (a, b) {
```

Comment:
This could use a comment describing what it does.  For example, what is ponderation?